pkg/pod/entrypoint_lookup_impl.go

```diff
@@ -18,14 +18,14 @@ package pod
 
 import (
 	"fmt"
-
 	"github.com/google/go-containerregistry/pkg/authn"
 	"github.com/google/go-containerregistry/pkg/authn/k8schain"
 	"github.com/google/go-containerregistry/pkg/name"
 	v1 "github.com/google/go-containerregistry/pkg/v1"
 	"github.com/google/go-containerregistry/pkg/v1/remote"
 	lru "github.com/hashicorp/golang-lru"
 	"k8s.io/client-go/kubernetes"
+	"runtime"
```

Comment:
technically `goimports` wants this up under `fmt` with a line of whitespace before the `github.com` imports.

Reply:
That's because of `gofmt`  - https://github.com/golang/go/issues/37719. Do you want me to change ordering? For this repo, we use `goimports` tool?

Reply:
`gofmt` moved it down because you removed the blank line.  May as well stick it back up at the top and add the newline while we are waiting on CLA 😅 

Sort of annoying that different go tools futz with the same parts, but in inconsistent ways...  thanks 🙏